project/Build.scala

```diff
@@ -1423,7 +1423,7 @@ object Build {
             .add(OutputDir("scaladoc/output/reference"))
             .add(SiteRoot(s"${temp.getAbsolutePath}/docs"))
             .add(ProjectName("Scala 3 Reference"))
-            .add(ProjectVersion("3.1.2")) // TODO: Change that later to the current version tag. (This must happen on first forward this branch to stable release tag)
+            .add(ProjectVersion("3.1.3")) // TODO: Change that later to the current version tag. (This must happen on first forward this branch to stable release tag)
```

Comment:
I'm not really sure how to understand that. If I'm not mistaken reference documentation is re-published every time something changes inside of it. If we use the `baseVersion` here, documentation will always display some unreleased RC version. `previousDottyVersion` should work, as it is changed on `main` just after every release, but I'm not yet sure.

Reply:
I think you should consider adding more documentation about how the documentation itself is published because it is not so simple and we get confused. As far as I understand, only the branch `language-reference-stable` is periodically built to produce the Scala 3 reference at https://docs.scala-lang.org/scala3/reference. I think there should be a step in the release process that consists of merging `main` to `language-reference-stable` so that `baseVersion` (or maybe another variable?) gets updated in the `language-reference-stable` branch with the right number. It could be done differently, maybe…

Reply:
After merging this PR, we should merge main into language-reference-stable and everything should be fine

Reply:
I now understand how it should be handled. Next time we will take care of the doc version during the release procedure.

To have a short-time fix, I will change the base of this PR to the stable release branch. This way there is no risk that we may contaminate stable documentation with things that are meant for 3.2.